From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Mon, 29 Jan 2018 12:26:43 +0000
Subject: Allow reading time configuration from /etc/writable,
 as we have it on the phone.

This is an Ubuntu specific tweak, so no need to forward upstream.

Forwarded: not-needed
Bug-Ubuntu: https://launchpad.net/bugs/1227520
---
 profiles/apparmor.d/abstractions/base                   |    1 +
 profiles/apparmor.d/abstractions/ubuntu-browsers.d/java |    2 ++
 profiles/apparmor/profiles/extras/firefox               |    1 +
 3 files changed, 4 insertions(+)

--- apparmor-4.1.0~beta1.orig/profiles/apparmor.d/abstractions/base
+++ apparmor-4.1.0~beta1/profiles/apparmor.d/abstractions/base
@@ -33,6 +33,7 @@
   @{etc_ro}/locale.alias       r,
   @{etc_ro}/localtime          r,
   @{etc_rw}/localtime          r,
+  /etc/writable/localtime        r,
   /usr/share/locale-bundle/**    r,
   /usr/share/locale-langpack/**  r,
   /usr/share/locale/             r,
--- apparmor-4.1.0~beta1.orig/profiles/apparmor.d/abstractions/ubuntu-browsers.d/java
+++ apparmor-4.1.0~beta1/profiles/apparmor.d/abstractions/ubuntu-browsers.d/java
@@ -39,6 +39,7 @@
     /etc/lsb-release r,
     /etc/ssl/certs/java/* r,
     /etc/timezone r,
+    /etc/writable/timezone r,
 
     @{PROC}/@{pid}/ r,
     @{PROC}/@{pid}/fd/ r,
@@ -86,6 +87,7 @@
     /etc/lsb-release r,
     /etc/ssl/certs/java/* r,
     /etc/timezone r,
+    /etc/writable/timezone r,
 
     @{PROC}/@{pid}/ r,
     @{PROC}/@{pid}/fd/ r,
